bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -12,6 +12,7 @@ class PyNeurodamus(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus-py.git"
 
     version("develop", branch="main", submodules=True)
```

Comment:
Should we remove the `submodules=True` from the `develop` version as well?

Reply:
good point. I will remove that